apps/web/src/hooks/usePoolAvgInfo.ts

```diff
@@ -34,8 +35,13 @@ const defaultInfo: Info = {
 }
 
 export function usePoolAvgInfo({ address = '', numberOfDays = 7, chainId }: UsePoolAvgInfoParams) {
+  const router = useRouter()
+  const isInFarmsPage = router.pathname.includes('/farms')
```

Comment:
we better not depend on the route path. Can we use param to explicitly control the behavior instead?